datasafe-simple-adapter/datasafe-simple-adapter-impl/src/main/java/de/adorsys/datasafe/simple/adapter/impl/DFSRelativeProfileRetrievalServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package de.adorsys.datasafe.simple.adapter.impl;
+
+import de.adorsys.datasafe.directory.api.config.DFSConfig;
+import de.adorsys.datasafe.directory.api.profile.dfs.BucketAccessService;
+import de.adorsys.datasafe.directory.api.types.CreateUserPrivateProfile;
+import de.adorsys.datasafe.directory.api.types.CreateUserPublicProfile;
+import de.adorsys.datasafe.directory.api.types.UserPrivateProfile;
+import de.adorsys.datasafe.directory.api.types.UserPublicProfile;
+import de.adorsys.datasafe.directory.impl.profile.operations.UserProfileCache;
+import de.adorsys.datasafe.directory.impl.profile.operations.actions.ProfileRetrievalServiceImpl;
+import de.adorsys.datasafe.directory.impl.profile.serde.GsonSerde;
+import de.adorsys.datasafe.encrypiton.api.types.UserID;
+import de.adorsys.datasafe.encrypiton.api.types.UserIDAuth;
+import de.adorsys.datasafe.encrypiton.api.types.keystore.ReadKeyPassword;
+import de.adorsys.datasafe.storage.api.actions.StorageCheckService;
+import de.adorsys.datasafe.storage.api.actions.StorageReadService;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DFSRelativeProfileRetrievalServiceImpl extends ProfileRetrievalServiceImpl {
+
+    DFSRelativeProfileRetrievalServiceImpl(DFSConfig dfsConfig, StorageReadService readService,
+                                                  StorageCheckService checkService, BucketAccessService access,
+                                                  GsonSerde serde, UserProfileCache userProfileCache) {
+        super(dfsConfig, readService, checkService, access, serde, userProfileCache);
```

Comment:
You can have 1-arg constructor and call super with all nulls (you do not use super)